### PR TITLE
Track whenever an OAuth client does something with the user

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
This is prep for https://github.com/mozilla/id.webmaker.org/issues/97

File: app/http/middleware.js [hunk 1]
```diff
@@ -96,6 +96,19 @@ module.exports.updateUser = function (User) {
   };
 };
 
+module.exports.createOauthLogin = function (User) {
+  return function (req, res, next) {
+    console.log(req.body);
```
Comment (cadecairos) [inline app/http/middleware.js]: remoooove

**Comment by adamlofting** [comment]
Just to note that this looks good to me WRT the data being captured.

**Comment by secretrobotron** [comment]
@jbuck ping. Can you make sure this is in working order? If it's good, I'll fix it up to merge.

**Comment by adamlofting** [comment]
@jbuck @secretrobotron Am I right thinking this didn't land?

**Comment by secretrobotron** [comment]
Almost certainly. @jbuck can you confirm?

**Comment by jbuck** [comment]
I actually did land this: https://github.com/mozilla/login.webmaker.org/commit/5dd05ada0fc1003368256ca2dc36ca307aad55f8

But it's kinda useless because it'll only record a login when the user actually signs in with username/password, not if they're already signed in.

**Pull request closed without merging**